projects/llm_d/runtime/llmd_runtime.py

```
@@ -0,0 +1,695 @@
+
+
+def run_command(
+    args: Iterable[str],
+    *,
+    check: bool = True,
+    capture_output: bool = True,
+    input_text: str | None = None,
+) -> subprocess.CompletedProcess[str]:
+    cmd = [str(arg) for arg in args]
+    LOGGER.info("run: %s", " ".join(shlex.quote(arg) for arg in cmd))
+    result = subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)